misc/packaging/ansible-runner-service.spec

```diff
@@ -75,6 +75,10 @@ install -m 0644 ./LICENSE.md %{buildroot}%{_docdir}/ansible-runner-service
 %{_docdir}/ansible-runner-service/*
 
 %changelog
+* Thu Jun 4 2020 Martin Necas <mnecas@redhat.com> 1.0.3-1
+- Add logrotate
```

Comment:
We haven't changed anything there, right? If so, we should mention that there was no change